RubberduckTests/Grammar/AttributesTests.cs

```diff
@@ -0,0 +1,1692 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using NUnit.Framework;
+using Rubberduck.Inspections.Concrete;
+using Rubberduck.Parsing.Symbols;
+using RubberduckTests.Mocks;
+
+namespace RubberduckTests.Grammar
+{
+    [TestFixture]
+    public class AttributesTests
```

Comment:
It seems to me that a number of tests, especially those dealing with different placement of the attributes for the function/sub/property{let/get/set} could have been collapsed into a single data-driven method since the `inputCode` is the only thing that's different.

Reply:
I did not use data-driven tests because the input code would be a giant parameter. It is more than half the size of the test methods. Putting that large parameters into the square brackets above the test would be unwieldy and putting them into a separate test source would move them too far away from the actual test.

Moreover, with data-driven tests, I do not get different names for the tests.